content-webapp/src/main/webapp/js/newsUtils.js

```diff
@@ -30,4 +30,16 @@ export function convertDate(date) {
 
 export function pad(n) {
   return n < 10 && `0${n}` || n;
-}
\ No newline at end of file
+}
+
+export const NewsUpdateType = {
+  CONTENT: 'content',
+  POSTING_AND_PUBLISHING: 'postingAndPublishing',
+  SCHEDULE: 'schedule'
+};
+
+export const NewsObjectType = {
```

Comment:
```suggestion
export const newsObjectType = {
```
I think that you can declare that inti newsConstants.js instead of newsUtils.js